README.md

```diff
@@ -49,3 +49,26 @@ connector, as long as the user has proper access to those tables.
 
 Creating a Destination connector will fail if the table does not exist or if the user does not have permission to work
 with the specified table.
+
+## Source
+
+The ClickHouse Source allows you to move data from the ClickHouse table to any Conduit Destination. When the connector
```

Comment:
`to any Conduit Destination ` - unfortunately it is can be not true) . I don't about do we need write this assertion

Reply:
Should I use `some` instead of `any`? Or do you mean I should remove this sentence entirely?

Reply:
maybe just remove word any it is will be looks like:
The ClickHouse Source allows you to move data from the ClickHouse table to Conduit Destination connectors

Reply:
Done.